requirements/prod

```diff
@@ -1,3 +1,3 @@
-jsonpickle
-tornado
-click
+jsonpickle==0.9.2
+tornado==4.2.1
+arteria==0.1.0
```

Comment:
The installation just stalled yesterday when I tried something like that, but I can give it an other shot.